devdocs/settingUp.md

```diff
@@ -53,6 +55,10 @@ Important: When a version is specified, please install that version instead of t
    * `src/main/webapp/WEB-INF/appengine-web.xml`<br>
    Create it using `appengine-web.template.xml`. 
    For now, property values can remain as they are.
+   * `.settings/com.google.gdt.eclipse.core.prefs`<br>
+   Create it using `com.google.gdt.eclipse.core.template.prefs`.
+   Change all `*` in `jarsExcludedFromWebInfLib` to your TEAMMATES project folder, e.g. `C:/TEAMMATES`.
```

Comment:
Rephrase like this: In the newly created `com.google.gdt.eclipse.core.prefs` file, replace the `*` in value of `jarsExcludedFromWebInfLib` to your TEAMMATES project folder.
e.g. `jarsExcludedFromWebInfLib=*/src/test/resources/lib/appengine/appengine-api-labs.jar` becomes `jarsExcludedFromWebInfLib=C:/TEAMMATES/src/test/resources/lib/appengine/appengine-api-labs.jar`